src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="post">
+      <input
+        type="text"
+        class="input-big"
+        data-qa="big"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form action="#" method="post">
+      <input
+        type="text"
+        id="input-small"
+        class="input-small input-small2"
```

Comment:
```suggestion
        class="input-small"
```
Клас input-small2 ніде не використовується